File: RubyScript/src/org/knime/ext/jruby/RubyScriptNodeFactory01.java
package org.knime.ext.jruby;

/**
 * <code>NodeFactory</code> for the "RubyScript" Node.
 * 
 *
 * @author 
 */
public class RubyScriptNodeFactory01
        extends RubyScriptNodeFactory {

    /**
     * {@inheritDoc}
     */
    @Override
    public RubyScriptNodeModel createNodeModel() {
        return new RubyScriptNodeModel(0,1);
    }
}

File: RubyScript/src/org/knime/ext/jruby/RubyScriptNodeFactory01.scala
package org.knime.ext.jruby

/**
 * <code>NodeFactory</code> for the "RubyScript" Node.
 * 
 * @author rss
 *
 */
class RubyScriptNodeFactory01 extends RubyScriptNodeFactory {

  /* (non-Javadoc)
   * @see org.knime.ext.jruby.RubyScriptNodeFactory#createNodeModel()
   */
  override def createNodeModel(): RubyScriptNodeModel = {
    setModel(new RubyScriptNodeModel(0, 1, false))
  }
}
